okta-idx-kotlin/src/main/java/com/okta/idx/kotlin/dto/IdxCapability.kt

```diff
@@ -181,6 +184,85 @@ class IdxTotpCapability internal constructor(
     }
 }
 
+/**
+ * Describes the WebAuthn registration capability associated with an [IdxAuthenticator].
+ *
+ * @property activationData The JSON string containing the public key credential creation options for WebAuthn activation.
+ */
+class IdxWebAuthnRegistrationCapability internal constructor(
+    private val activationData: String,
+) : IdxAuthenticator.Capability {
+
+    /**
+     * Returns the public key credential creation options as a JSON string.
+     *
+     * @param rpId Optional relying party ID. If provided and not blank, overrides the default `rp.id` in the activation data if present.
+     * @return A [Result] containing the JSON string with the `rp.id` field set to [rpId] if specified, or the original activation data otherwise.
+     */
+    fun publicKeyCredentialCreationOptions(rpId: String? = null): Result<String> = runCatching {
```

Comment:
I generally prefer to see descriptive variable & argument names, especially since developers may not understand what some acronyms mean.  In the Swift SDK, I named this [`relyingPartyIdentifier`](https://github.com/okta/okta-mobile-swift/blob/master/Sources/OktaIdxAuth/Capabilities/WebAuthnRegistrationCapability.swift#L44), to make it clearer what the value is used for.